geth/node/rpc_test.go

```diff
@@ -20,9 +33,70 @@ type RPCTestSuite struct {
 
 func (s *RPCTestSuite) SetupTest() {
 	require := s.Require()
-	s.NodeManager = node.NewNodeManager()
+
+	s.NodeManager = proxy.NewRPCRouter(node.NewNodeManager())
+	require.NotNil(s.NodeManager)
+	require.IsType(&proxy.RPCRouter{}, s.NodeManager)
+}
+
+func (s *RPCTestSuite) TestUpstreamCallRPC() {
```

Comment:
What's the difference with `TestRPCClientConnection` in `geth/proxy/rpc_test.go`?